packages/models-library/src/models_library/groups.py

```diff
@@ -0,0 +1 @@
+EVERYONE_GROUP_ID = 1
```

Comment:
this could be a bit dangerous. if I remember well the everyone group is 99% time the number of 1. but in theory could be different. there is a group type, and this is the way to 100% get it right. maybe I'm too cautious there.

Reply:
Yes, as you can see from the code below, we are already using this assumption in some query, so I guess for now it might be ok. Otherwise, I would need to propagate the group type from the catalog, currently, I am reusing as much code as possible, unfortunately, the group type is not provided in the ServiceAccessRightsAtDB model.